src/Parser/Core/Modules/Items/BFA/Dungeons/GalecallersBoon.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import Analyzer from 'Parser/Core/Analyzer';
+import { formatPercentage, formatNumber } from 'common/format';
+import { calculateSecondaryStatDefault } from 'common/stats';
+
+/**
+ * Galecaller's Boon
+ * Place a ward on the ground for 10 sec which increases your Haste by [x] and your Speed by [x] while you stand within it.
+ *
+ * Example: https://www.warcraftlogs.com/reports/BhLHrn1PzQRJ6XVZ/#fight=6&source=8&type=auras&ability=268311
+ */
+class GalecallersBoon extends Analyzer {
+  statBuff = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrinket(ITEMS.GALECALLERS_BOON.id);
+
+    if (this.active) {
+      this.statBuff = calculateSecondaryStatDefault(310, 917, this.selectedCombatant.getItem(ITEMS.GALECALLERS_BOON.id).itemLevel);
```

Comment:
the haste here should *probably* be a named constant

Reply:
Just stole it off existing Azerite traits 😅